keps/sig-node/3573-device-plugin/README.md

```diff
@@ -779,6 +781,8 @@ No
 
 ###### How can an operator determine if the feature is in use by workloads?
 
+1. The operator can look at `device_plugin_registration_total` and `device_plugin_alloc_duration_seconds`
+   metrics.
```

Comment:
especially the second metric, if nonzero, tells that device allocation were performed.
Additionally, perhaps replacing the last bullet below: check if the pod spec contains request of non-native (cpu, memory, hugepages) resources?